h2o-r/tests/testdir_algos/isoforextended/runit_isoforextended_smoke.R

```diff
@@ -0,0 +1,19 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+
+
+test.ExtendedIsolationForest.smoke <- function() {
+    set.seed(1234)
+    N = 1e4
+    random_data <- data.frame(
+        x = c(rnorm(N, 0, 0.5), rnorm(N*0.05, -1.5, 1)),
+        y = c(rnorm(N, 0, 0.5), rnorm(N*0.05,  1.5, 1))
+    )
+    random_data.hex <- as.h2o(random_data)
+    
+    exisofor.model <- h2o.extendedIsolationForest(training_frame = random_data.hex)
+    expect_equal(as.character(class(exisofor.model)), "H2OAnomalyDetectionModel")
```

Comment:
The same comment as in the Python test, try to compare result values from the `predict` method.

Reply:
Done